presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -114,6 +115,11 @@ public ProjectNode project(Assignments assignments, PlanNode source)
         return new ProjectNode(idAllocator.getNextId(), source, assignments);
     }
 
+    public MarkDistinctNode markDistinct(Symbol markerSymbol, List<Symbol> distinctSymbols, Optional<Symbol> hashSymbol, PlanNode source)
```

Comment:
Sure, if you like I'll do that.  Ideally, we'd have builders for plan nodes automatically generated from their declarations, along with things like PlanVisitor.